pkg/lib/cockpit-components-modifications.jsx

```diff
@@ -37,7 +37,7 @@ const _ = cockpit.gettext;
  * Enables showing shell and ansible script. Shell one is mandatory and ansible one can be omitted.
  *
  */
-class ModificationsExportDialog extends React.Component {
+export class ModificationsExportDialog extends React.Component {
```

Comment:
Accident? The commit message doesn't explain making it a public API, and it's also unrelated. The `Modifications` component is meant to be the API.

Reply:
Ah, that's a leftover from my kdump experiment.

Reply:
This is still here. Probably a rebase error, as the next commit removes it.